easybuild/easyconfigs/e/EMAN/EMAN-daily.eb

```diff
@@ -0,0 +1,31 @@
+easyblock = 'PackedBinary'
+
+name = 'EMAN'
+version = 'daily'
```

Comment:
EMAN has proper version numbers, so use them?

cfr. http://ncmi.bcm.edu/ncmi/software/counter_222/software_122

Also, the official software name seems to be `EMAN2`